azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/utils/Utils.java

```diff
@@ -176,17 +210,16 @@ public static boolean isExecutableJar(File file) {
         };
     }
 
-    public static <T> T executeCallableWithPrompt(Callable<T> callable, String prompt, int timeOutInSeconds) {
-        final ExecutorService executorService = Executors.newFixedThreadPool(2);
-        final Future<T> future = executorService.submit(callable);
-        final Future<T> wrappedFuture = executorService.submit(getWrappedCallable(prompt, future));
-        try {
-            return wrappedFuture.get(timeOutInSeconds, TimeUnit.SECONDS);
-        } catch (Exception e) {
-            wrappedFuture.cancel(true);
-            return null;
-        } finally {
-            executorService.shutdown();
+    private static File getRunnableArtifactFromFiles(List<File> files) throws MojoExecutionException {
+        final List<File> runnableArtifacts = files.stream().filter(Utils::isExecutableJar).collect(Collectors.toList());
+        if (runnableArtifacts.size() == 1) {
+            return runnableArtifacts.get(0);
+        } else if (runnableArtifacts.size() == 0) {
```

Comment:
isEmpty

Reply:
replace the if -> else if -> else if to if if if

Reply:
Fixed.